apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -32,12 +32,13 @@
 
 #define MICROSECOND 1000000
 #define BUF_LEN 32
-#ifdef CONFIG_ARCH_CHIP_STM32
-#define DEVPATH "/dev/ttyS1"
-#else                            // artik
-#define DEVPATH "/dev/ttyS2"
+
+#if !defined(CONFIG_SYSIO_ITC_UART_FILE_PATH)
+#error To run system I/O testcase, kindly set file path of uart on menuconfig.
```

Comment:
Please add the description, which config should be set.

Reply:
Done